client/Combatant/Combatant.ts

```diff
@@ -54,6 +55,9 @@ export class Combatant {
   public Hidden = ko.observable(false);
   public RevealedAC = ko.observable(false);
 
+  public CombatRounds = ko.observable(0);
```

Comment:
The properties and methods that were added to Combatant here should all be encapsulated into a class, possibly named `CombatTimer`.

Reply:
Since `CombatTimer` needs to track rounds as well, there is some overlap with `EncounterFlow.RoundCounter()`. I played around with removing `RoundCounter` and replacing it with a `pureComputed` wrapper for `CombatTimer.ElapsedRounds()` - but things got messy, and it's honestly a bit out of my range of expertise. In particular, there are a few places that write to `RoundCounter`, (like loading saved encounters).

There is also significant overlap between `CombatTimer` and `TurnTimer`.

Reply:
I'll do some minor refactoring to accommodate EncounterState and CombatantState.

Reply:
Sounds good. You could merge CombatTimer and TurnTimer if one completely obsoletes the other.

Reply:
I still think the properties and methods that are added to Combatant here could be encapsulated in their own CombatantTimer class. Not required for merge, but a nice-to-have.